src/websocket_service/handlers/notify_watch_subscriptions.rs

```diff
@@ -180,17 +180,32 @@ pub async fn collect_subscriptions(
         get_subscriptions_by_account(account, postgres).await?
     };
 
-    let subscriptions = subscriptions
-        .into_iter()
-        .map(|sub| NotifyServerSubscription {
-            app_domain: sub.app_domain,
-            app_authentication_key: sub.authentication_public_key,
-            sym_key: sub.sym_key,
-            account: sub.account,
-            scope: sub.scope.into_iter().collect(),
-            expiry: sub.expiry.timestamp() as u64,
-        })
-        .collect::<Vec<_>>();
+    let subscriptions = {
+        let try_subscriptions = subscriptions
+            .into_iter()
+            .map(|sub| {
+                fn wrap(sub: SubscriberWithProject) -> Result<NotifyServerSubscription> {
+                    Ok(NotifyServerSubscription {
+                        app_domain: sub.app_domain,
+                        app_authentication_key: format!(
+                            "did:key:{}",
+                            DecodedClientId(decode_key(&sub.authentication_public_key)?)
```

Comment:
Something smells. It's not DecodedClientId, It's encoded ED25519 key. Do we have tests for this?